src/main/scala/org/apache/mesos/chronos/scheduler/mesos/MesosTaskBuilder.scala

```diff
@@ -36,7 +36,7 @@ class MesosTaskBuilder @Inject()(val conf: SchedulerConfiguration) {
       .setName(name)
       .setType(Value.Type.SCALAR)
       .setScalar(Value.Scalar.newBuilder.setValue(value))
-      .setRole("*")
+      .setRole(conf.mesosRole())
```

Comment:
@brndnmtthws Resource was taking * as the hardcoded value. Now the conf value set in  --mesos-role is used. I have tested in mesos using multiple chronos frameworks and various roles. My cluster now does not have * roles. Also using Resource.newBuilder we can only setRole. Its's not possible to add multiple roles.